docs/technical_reference/costing/ion_exchange.rst

```diff
@@ -6,10 +6,29 @@ Costing Method Parameters
 
 The following parameters are constructed for the unit on the FlowsheetCostingBlock (e.g., `m.fs.costing.ion_exchange`) when applying the `cost_ion_exchange` costing method in the ``watertap_costing_package``:
 
-.. csv-table::
-   :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "parameter_name", "1", ":math:`\text{dimensionless}`"
+.. csv-table::
+   :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units", "Notes"
+
+   "Anion exchange resin cost", ":math:`c_{res}`", "``anion_exchange_resin_cost``", "205", ":math:`\text{\$/}\text{ft}^{3}`", "Assumes strong base polystyrenic gel-type Type II. From EPA-WBS cost model."
```

Comment:
I like how you formatted the parameter names in this table. Perhaps I should update the automation script so that the templates automatically use this format. Thoughts @adam-a-a @hunterbarber or others? It would mean going back and updating all the existing documentation tho...

Reply:
Yes I like using the monospaced font for anything that I could find in a Python file/object, just hits the eyes better imo. I am happy to change it if there is consensus either way.

Reply:
I don't think anyone would be opposed to this idea, so I can address it in #1394